android/app/src/main/kotlin/org/getlantern/lantern/model/ProUser.kt

```diff
@@ -7,14 +7,14 @@ import org.joda.time.Months
 data class ProUser(
     val userId: Long,
     val token: String,
-    val referral: String,
-    val email: String,
-    val userStatus: String,
-    val code: String,
-    val subscription: String,
-    val expiration: Long,
-    val devices: List<Device>,
-    val userLevel: String,
+    val referral: String = "",
+    val email: String = "",
+    val userStatus: String = "",
+    val code: String = "",
+    val subscription: String = "",
+    val expiration: Long = 0,
+    val devices: List<Device> = mutableListOf<Device>(),
+    val userLevel: String = "",
```

Comment:
It looks like the app isn't crashing anymore without these changes, but I think setting default values here is helpful to avoid the issue in the future

Reply:
I agree with you 100%, Most values should be nullable or have default values to avoid crashes.

Reply:
Let's just put out a new release with these changes then.

Reply:
@atavism Can we wait till tomorrow? I think I might caught bug,